<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="lcdnumber_log_file">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_elf_file">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="digitCount">
     <number>9</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_read_log_file">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Read log file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_read_elf_file">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Read elf file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add_number">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>440</y>
      <width>711</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_log_file">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>1081</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_elf_file">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>1081</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>741</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Manual_Number_checkBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Manual Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Manualk_Number_lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionOpen_Log_File"/>
    <addaction name="actionOpen_Elf_File"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Log_File">
   <property name="text">
    <string>Open Log File</string>
   </property>
  </action>
  <action name="actionOpen_Elf_File">
   <property name="text">
    <string>Open Elf File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
